chrome/browser/download/download_ui_controller.cc

```diff
@@ -159,6 +162,17 @@ void DownloadUIController::OnDownloadUpdated(content::DownloadManager* manager,
 #if !defined(OS_ANDROID)
   content::WebContents* web_contents = item->GetWebContents();
   if (web_contents) {
```

Comment:
It's download url which is http://download.microsoft.com/download/F/7/3/F7395EE6-5642-4AB9-A881-786D0350E88D/Skills_Development_White_Paper_2009.doc based on following html.

 <a target="_blank" href="http://download.microsoft.com/download/F/7/3/F7395EE6-5642-4AB9-A881-786D0350E88D/Skills_Development_White_Paper_2009.doc"> Click Me(Doc) </a>